api-doc/api.raml

```diff
@@ -0,0 +1,21 @@
+#%RAML 1.0
+title: eLife's Libero walking skeleton
+
+description: |
+    Example of a composed Libero API schema.
+
+baseUri: http://localhost:8085
+
+resourceTypes:
+    articles: !include articles.raml
+
+types:
+    ArticleVersionFront:
+        type: !include http://api-dummy-nginx:80/schemas/elife/article/front.rng
```

Comment:
Reading the RAML spec, I think this is technically invalid. Seems ok to process JSON Schema or XML Schema, but doesn't allow for other things. raml2html emits a warning (actually in raml2obj) and just outputs the RELAX NG instead.

Reply:
Related: https://github.com/raml-org/raml-spec/issues/117